ops/charm.py

```diff
@@ -893,7 +893,92 @@ class ContainerMeta:
 
     Attributes:
         name: Name of container (key in the YAML)
+        mounts: :class:`ContainerStorageMeta` mounts available to the container
     """
-
     def __init__(self, name, raw):
         self.name = name
+        self._mounts = {}
+
+        # This is not guaranteed to be populated/is not enforced yet
+        if raw:
+            self._populate_mounts(raw.get('mounts', []))
+
+    @property
+    def mounts(self) -> typing.Dict:
+        """An accessor for the mounts in a container.
+
+        Dict keys match key name in :class:`StorageMeta`
+
+        Example::
```

Comment:
```suggestion
        Example:
```

Reply:
This one is actually required by Sphinx+reStructuredText to get a block literal. No single `:` if we want it to render properly

Reply:
Doh. Brainfart.